compiler/src/iree/compiler/Codegen/LLVMGPU/test/nvvm_mma_sync_pipeline_test.mlir

```diff
@@ -146,16 +146,40 @@ hal.executable @mma_fused_f32 {
 //          CHECK:   nvvm.cp.async.commit.group
 //          CHECK:   nvvm.cp.async.wait.group 2
 //  CHECK-COUNT-1:   nvvm.ldmatrix{{.*}} : (!llvm.ptr<3>) -> !llvm.struct<(i32, i32, i32, i32)>
-//  CHECK-COUNT-4:   llvm.extractvalue{{.*}} : !llvm.struct<(i32, i32, i32, i32)> 
+//  CHECK-COUNT-4:   llvm.extractvalue{{.*}} : !llvm.struct<(i32, i32, i32, i32)>
+//  CHECK-COUNT-1:   llvm.load{{.*}} : !llvm.ptr<3> -> f32
+//  CHECK-COUNT-1:   llvm.insertvalue{{.*}} : !llvm.array<2 x vector<1xf32>>
```

Comment:
How can I check these two instructions {llvm.load, llvm.insertvalue} are repeated 4 times without copy-pasting it four times?

Reply:
you have to put the two on the same line.

Either combine them that way or remove the `COUNT-1`